__tests__/commands/string.test.ts

```diff
@@ -0,0 +1,14 @@
+import { describe, expect, test } from '@jest/globals';
```

Comment:
It will be better to place this under utils/.

Reply:
describe, expect, and test belong to @jest/globals package, so we just call it out from @jest/globals package

Reply:
I mean that there is `/src/utils/string.ts`, so it will be better to follow that directory structure under test like `/__tests__/utils/string.test.ts`, too.

Reply:
I updated code here:
https://github.com/kintone-labs/kintone-cli/pull/119/commits/3a726c2c7023fc72eaa758e37ebec2898b3ba942

Please have a check.
Thank you!

Reply:
I cannot see the change in string.test.ts file's directory.
What is your intension of the comment above?

Reply:
This { describe, expect, test } belong to test, so I put it in test-helpers.
The test-helpers folder in __tests__ has the same function as the utils folder in src/utils
It's ok? Do you want to rename test-helpers to utils?

Reply:
I think it's ok with test-helpers.
My previous comment intention was that there is `src/utils/string.ts`, so please implement unit test for that file under `__tests/utils/string.test.ts` as same as commands.
Please confirm with PG members, too.

Reply:
I renamed test-helpers to utils:
https://github.com/kintone-labs/kintone-cli/pull/119/commits/ec2b3ffe0fa2df9ddde940c38aa7fece94852e6c

Reply:
I don't think you really understand what I want to say.
I will explain it again here.

There is `src/utils/string.ts`, so please implement a unit test for that file under `__tests/utils/string.test.ts` as same as `commands/`.
I think you deleted string.test.ts because it's not related to Auth commands and it's ok!

About test-helpers, it's only for testing and it's not related to `src/utils/`, so I think it's ok with the previous directory and file name `test-helpers/index.ts`.

Does it make sense?

Reply:
Please check the above comment.

Reply:
I saw the test directory structure has changed.